src/components/card-container/card-container.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+class CardContainer extends React.PureComponent {
+  render() {
+    const { props } = this;
+    const categoryID = props.category.path.split('#')[1];
+    return (
+      <div>
+        <a
+          href={props.category.path}
+          className="unprose mb12 block color-blue-on-hover"
+        >
+          <h2 className="pt60 txt-l" id={categoryID}>
```

Comment:
This component shouldn't determine its own top padding: the parent should lay it out in relation to other things.